ts/screens/authentication/IdpLoginScreen.tsx

```diff
@@ -96,32 +119,94 @@ const onNavigationStateChange = (
 class IdpLoginScreen extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
-
+    const nowMs = new Date().getTime();
     this.state = {
-      requestState: pot.noneLoading
+      requestState: pot.noneLoading,
+      startingMillis: nowMs as Millisecond,
```

Comment:
You are not using this variable to alter the rendering of the component. I think is better to store it as normal instance prop.

Reply:
Thank you, this is a bad usage of state (not relating to UI). I will apply your suggestion